pandas/core/arrays/datetimes.py

```diff
@@ -2158,10 +2158,16 @@ def _sequence_to_dt64ns(
             # Convert tz-naive to UTC
             # TODO: if tz is UTC, are there situations where we *don't* want a
             #  copy?  tz_localize_to_utc always makes one.
+            shape = data.shape
+            if data.ndim > 1:
+                shape = data.shape
```

Comment:
Why does this line need to be called again?

Reply:
good catch defining shape again is unnecessary